lib/modules/manager/mise/readme.md

```diff
@@ -0,0 +1,18 @@
+Renovate can update the [mise](https://mise.jdx.dev/configuration.html#mise-toml) `.mise.toml` file.
+
+Renovate's `mise` manager can version these tools:
+
+<!-- Autogenerate in https://github.com/renovatebot/renovate -->
+<!-- Autogenerate end -->
+
+<!-- prettier-ignore -->
+!!! warning
+    Only the _first_ version entry for each supported tool is managed!
+    This is due to `.mise.toml` supporting fallback versions.
```

Comment:
It feels we're missing a step in the explanation. Here's what I think when I read the current message:

1. Okay, so only the _first_ version for each tool is managed.
2. Looks like `.mise.toml` fallback versions cause this behavior.
3. But "why/how" does the fallback version "break" the way Renovate "sees/upgrades" versions?

Can you please improve the explanation, so the logical leap from "fallback version" to "broken Renovate" is clear? :wink: